GravatarApp/Share/ShareQRFullScreenView.swift

```diff
@@ -0,0 +1,102 @@
+import SwiftUI
+
+struct ShareQRFullScreenView<QRImage: View>: View {
+    @Binding var presentFullScreen: Bool
+    let topPadding: CGFloat
+    let imageURL: URL?
+    @Binding var forceRefresh: Bool
+    let windowWidth: CGFloat
+
+    @ViewBuilder
+    let qrImage: () -> QRImage
+
+    @State private var originalScreenBrightness: CGFloat = 0
+
+    var fullScreenQrWidth: CGFloat {
+        let padding = CGFloat.Global.contentHorizontalPadding
+        return min(
+            windowWidth - padding * 2,
+            500
+        )
+    }
+
+    var body: some View {
+        VStack(alignment: .leading) {
+            closeButton
+            Spacer()
+            centralContent
+            Spacer()
+            Spacer()
+        }
+        .background {
+            HeaderAvatarView(
+                imageURL: imageURL,
+                showLoading: false,
+                forceRefresh: $forceRefresh,
+                placeholderView: { EmptyView() }
+            )
+            .frame(maxHeight: .infinity)
+            .darkBlurImageStyle(width: windowWidth)
+        }
+        .environment(\.colorScheme, .dark)
+        .onAppear {
+            originalScreenBrightness = UIScreen.main.brightness
+            UIScreen.main.brightness = 1
```

Comment:
I felt like it's a bit too bright but I am not sure what the idea value would be

Reply:
This is a common pattern I see. And they use full brightness (I've checked 😅 )
We could try with 0.8 o 0.9 though. Since we expect this code to be read by other phones with relatively good cameras and no code reader machines which seem to be less quality. 🤔 

Maybe it's not that important and we skip it for now?